erddapy/core/interfaces.py

```diff
@@ -68,16 +67,15 @@ def to_xarray(
 
     url: URL to request data from.
     response: type of response to be requested from the server.
-    requests_kwargs: arguments to be passed to urlopen method.
+    requests_kwargs: arguments to be passed to urlopen method (including auth)
```

Comment:
We probably don't need this. We can add an example that has the `auth` dict inside the `requests_kwargs` dict to illustrate that though. What do you think?

```suggestion
    requests_kwargs: arguments to be passed to urlopen method
```

Reply:
I agree. I will add that example when I work on the docs again.